File: algoritmo_e_logica/python/q_7.py
# Crie uma função chamada calcularQuadrado que receba um número como parâmetro e retorne o quadrado desse número

def calcular_quadrado(numero):
    return numero ** 2

exemplos = [3, 7, 10]

for num in exemplos:
    print(f'O quadrado de {num} é {calcular_quadrado(num)}')

File: algoritmo_e_logica/javascript/q_7.js
// Crie uma função chamada calcularQuadrado que receba um número como parâmetro e retorne o quadrado desse número

function calcularQuadrado(numero) {
    return numero ** 2;
}

let exemplos = [3, 7, 10];

exemplos.forEach(num => {
    console.log(`O quadrado de ${num} é ${calcularQuadrado(num)}`);
});